src/backend/expungeservice/models/disposition.py

```diff
@@ -1,18 +1,60 @@
 from dataclasses import dataclass
 from datetime import date, datetime
 from typing import Optional
+from enum import Enum
 
 
+class DispositionStatus(str, Enum):
+    CONVICTED = "Convicted"
+    DISMISSED = "Dismissed"
+    NO_COMPLAINT = "No Complaint"
+    DIVERTED = "Diverted"
+    UNKNOWN = "Unknown"
+
 @dataclass(eq=False)
 class Disposition:
     date: Optional[date]
     ruling: str
+    status: DispositionStatus
 
     def __init__(self, date: Optional[str] = None, ruling: str = ""):
         self.date = self.__set_date(date)
         self.ruling = ruling
+        self.status = self.__set_status()
```

Comment:
Use `__post_init__` to set `self.status` as we eventually just want to remove this `__init__` field. See https://docs.python.org/3/library/dataclasses.html#post-init-processing.